packages/services/src/rewards.service.ts

```diff
@@ -594,6 +598,8 @@ export class RewardsService {
       boostAllocations,
       leaderBoard,
       window,
+      PrizePoolDecayRate,
```

Comment:
the feature is already implemented, it's just a matter of passing the value to `calculateRewardsForUsers`